### PR TITLE
Add styling to support full-width NumericInput

### LINKED ISSUE #618: Numeric Input Issues
Thanks for the new component! Three issues encountered with it:

- [ ] 1. It doesn't seem possible to set the size of the input using something like `style={{width: '100%'}}`

- [x] 2. I'm experience artifacts where the stepper is misaligned. See the following screenshot:

image

- [x] 3. It would be nice to be able to prevent the user entering non-numeric values like <input type='number'> does

### PR DESCRIPTION
**Pull request by cmslewis**
#### Checklist

- [ ] Include tests
- [x] Update documentation

#### Changes proposed in this pull request:

Add `pt-fixed` to the `pt-button-group` within `NumericInput`. This ensures only the `pt-input-group` will stretch when the `NumericInput` is styled as full-width via a `pt-fill` class.

#### Reviewers should focus on:

- Is there a good way to test this?
- **DONE** ~Do we want to add a note mentioning you can style `NumericInput` with `pt-fill`?~

### CODE REVIEW

**Comment by giladgray** [comment]
yes probably worth adding such a note.

**Comment by giladgray** [comment]
_maybe_ even a switch in the example, but that's probably overkill.

**Comment by cmslewis** [comment]
@giladgray @llorca - let me know if the new language about responsive numeric inputs looks good:
https://github.com/palantir/blueprint/pull/792/files#diff-5b3d3683592e535ebca92a2b671191cdR98

**Comment by llorca** [comment]
@cmslewis Maybe a full section for this isn't needed, but otherwise LGTM

File: packages/docs/src/styles/_sections.scss [hunk 1]
```diff
@@ -180,6 +180,14 @@
   }
 }
 
+#{section-id("components.forms.numeric-input")} {
+  // .pt-fill doesn't work out-of-the-box within .docs-react-example,
+  // because the latter is styled with display: flex.
+  .docs-react-example .docs-react-numeric-input-example {
+    width: 100%;
```
Comment (llorca) [inline packages/docs/src/styles/_sections.scss]: @giladgray is this legit?

Comment (giladgray) [inline packages/docs/src/styles/_sections.scss]: yeah makes sense cuz flex box

**Pull request merged**